modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ues/ui/integration/test/dashboard/AddDeleteDashboardTestCase.java

```diff
@@ -0,0 +1,99 @@
+        super(userMode);
+        this.dashboardTitle = dashboardTitle;
+    }
+
+    @DataProvider(name = "userMode")
+    private static Object[][] userModeProvider() {
+        return new Object[][]{{TestUserMode.SUPER_TENANT_ADMIN, DASHBOARD_TITLE1}, {TestUserMode.SUPER_TENANT_USER, DASHBOARD_TITLE2}};
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void setUp() throws XPathExpressionException, AxisFault, MalformedURLException, InterruptedException {
+        String backendURL = getUesContext().getContextUrls().getBackEndUrl();
+        resourceAdminServiceClient = new ResourceAdminServiceClient(backendURL, getCurrentUsername(), getCurrentPassword());
+        resourcePath = DASHBOARD_REGISTRY_BASE_PATH + dashboardTitle.toLowerCase();
+        UESUtil.login(getDriver(), getBaseUrl(), getCurrentUsername(), getCurrentPassword());
+    }
+
+    @Test(groups = "wso2.ues.dashboard", description = "Adding new dashboard")
+    public void testAddDashboardNew() throws MalformedURLException, XPathExpressionException {
+        getDriver().findElement(By.cssSelector("[href='create-dashboard']")).click();
+        getDriver().findElement(By.id("ues-dashboard-title")).clear();
+        getDriver().findElement(By.id("ues-dashboard-title")).sendKeys(dashboardTitle);
+        getDriver().findElement(By.id("ues-dashboard-description")).clear();
+        getDriver().findElement(By.id("ues-dashboard-description")).sendKeys(DASHBOARD_DESCRIPTION);
+        getDriver().findElement(By.id("ues-dashboard-create")).click();
+        getDriver().findElement(By.id("single-column")).click();
+        getDriver().findElement(By.cssSelector("a.navbar-brand.ues-tiles-menu-toggle")).click();
+        getDriver().findElement(By.cssSelector("i.fw.fw-dashboard")).click();
+        getWebDriverWait().until(ExpectedConditions.visibilityOfElementLocated(By.id(dashboardTitle)));
+        webElement = getDriver().findElement(By.id(dashboardTitle));
+
+        assertEquals(dashboardTitle, webElement.findElement(By.cssSelector("h2")).getText());
+        assertEquals(DASHBOARD_DESCRIPTION, webElement.findElement(By.cssSelector("p")).getText());
+    }
+
+    @Test(groups = "wso2.ues.dashboard", description = "Deleting added dashboard", dependsOnMethods = "testAddDashboardNew")
+    public void testDeleteDashboardNew() throws MalformedURLException, XPathExpressionException {
+        webElement = getDriver().findElement(By.id(dashboardTitle));
+        webElement.findElement(By.cssSelector("i.fw-delete")).click();
+        getDriver().findElement(By.cssSelector("span.ladda-label")).click();
+        assertFalse(BaseUITestCase.isElementPresent(getDriver(), By.id(dashboardTitle)), "successfully deleted dashboard" + dashboardTitle);
+    }
+
+    @AfterClass
+    public void tearDown() throws MalformedURLException, XPathExpressionException, RemoteException, ResourceAdminServiceExceptionException {
+        resourceAdminServiceClient.deleteResource(resourcePath);
```

Comment:
This has done by removing the code resourceAdminServiceClient.deleteResource(resourcePath); and trying to checks whether the resource exists or not to ensure the dashboard deleted successfully.